src/arrayMethodJoin.js

```diff
@@ -4,8 +4,20 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joinString = '';
```

Comment:
Дієсловами називаємо функції.
```suggestion
    let joinedString = '';
```

Reply:
Join  ніби дієслово) як і Joined  тільки в present simple

Reply:
`join string` - об'єднуй рядки.
`joined string` - з'єднаний рядок